src/couch_httpd_auth.erl

```diff
@@ -366,6 +366,19 @@ handle_session_req(#httpd{method='DELETE'}=Req, _AuthModule) ->
 handle_session_req(Req, _AuthModule) ->
     send_method_not_allowed(Req, "GET,HEAD,POST,DELETE").
 
+extract_username(Form) ->
+    try couch_util:get_value("name", Form) of
```

Comment:
You miss the question about case of try/catch from previous response (:
There is no such case for this. To prove that let's take a look on L278-288 above.
`Form` expected to be a list, otherwise `couch_util:get_value` will raise `badarg` error. But, it couldn't be non-list.
If Content-Type is `application/x-www-form-urlencoded` then `mochiweb_util:parse_qs/1` called which returns a list.
If Content-Type is `application/json` then we match decoded json against `{Pairs}`. It payload is not a JSON object, we'll have `badmatch` here (btw, that's what is better to gracefully handle instead). Otherwise `Form` will be a list.
For every else Content-Type empty list is assignment. 

Any other case I miss when try/catch may be useful here?